src/raft/raft.rs

```diff
@@ -191,6 +192,9 @@ pub struct Raft<T: Storage> {
 
     /// tag is only used for logging
     tag: String,
+
+    /// Record the last instant of each peer's heartbeat response.
+    pub last_heartbeats: HashMap<u64, Instant>,
```

Comment:
why not put this in progress? 
But I prefer keeping these in peer, not in raft. @BusyJay